consensus/istanbul/backend/api.go

```diff
@@ -474,6 +477,90 @@ func (api *APIExtension) GetBlockWithConsensusInfoByHash(blockHash common.Hash)
 	return api.makeRPCBlockOutput(block, cInfo, block.Transactions(), receipts), nil
 }
 
+func (api *APIExtension) GetBlsInfos(number rpc.BlockNumber) (map[string]interface{}, error) {
+	kip113Addr, err := api.GetActiveAddressFromRegistry(system.Kip113Name, number)
+	if err != nil {
+		return nil, err
+	}
+
+	var bn *big.Int
```

Comment:
Please consider the below to make the variable init shorter.

```
	bn := big.NewInt(number.Int64())
	if number == ... {}
```

Reply:
Thanks. Applied at https://github.com/klaytn/klaytn/pull/2111/commits/4ac64d4f91554c2b35f66f87c13820f11d7e34a6